src/master-playlist-controller.js

```diff
@@ -1029,14 +1034,16 @@ export class MasterPlaylistController extends videojs.EventTarget {
           return;
         }
         const excludeUntil = playlist.excludeUntil;
-
         // a playlist cannot be reincluded if it wasn't excluded to begin with.
+
         if (typeof excludeUntil !== 'undefined' && excludeUntil !== Infinity) {
           reincluded = true;
           delete playlist.excludeUntil;
         }
       });
 
+      this.blacklistedPlaylistReloads++;
```

Comment:
Looks like this value is only ever incremented here, I'd imagine that on a successful playback it should be reset back to 0? I'm not exactly sure where it should go, though.

Reply:
Updated so that we now reset the value on `appendsdone`. In case of an error, we `return` before appendsdone is fired. @gesinger can you confirm that `appendsdone` is an appropriate place to reset the number of final rendition retries?